util/git/client.go

```diff
@@ -318,11 +328,18 @@ func (m *nativeGitClient) Fetch(revision string) error {
 	}
 
 	var err error
-	if revision != "" {
-		err = m.runCredentialedCmd("git", "fetch", "origin", revision, "--tags", "--force")
-	} else {
-		err = m.runCredentialedCmd("git", "fetch", "origin", "--tags", "--force")
+
+	err = m.fetch(revision)
+	if err != nil {
+		if strings.Contains(err.Error(), "git remote prune origin") {
```

Comment:
The full error message, including line breaks, is:

```
`git fetch origin --tags --force` failed exit status 1: error: cannot lock ref 'refs/remotes/origin/test/foo/bar': 'refs/remotes/origin/test/foo' exists; cannot create 'refs/remotes/origin/test/foo/bar'
From file:///var/folders/fw/grysx2h56354vh4_30px6_r80000gp/T/250030459
! [new branch]      test/foo/bar -> origin/test/foo/bar  (unable to update local ref)
error: some local refs could not be updated; try running
'git remote prune origin' to remove any old, conflicting branches
```

Is it adequate to match against just `git remote prune origin`?

Reply:
What if we match against `try running 'git remote prune origin'`? A little more risk of breakage, but clearer-reading code.